OpenEdXMobile/src/main/java/org/edx/mobile/profiles/UserProfileFragment.java

```diff
@@ -194,6 +201,18 @@ public void showError(@NonNull Throwable error) {
                 viewHolder.profileBodyContent.setVisibility(View.GONE);
                 viewHolder.contentError.getRoot().setVisibility(View.VISIBLE);
                 viewHolder.contentError.contentErrorText.setText(ErrorUtils.getErrorMessage(error, getContext()));
+                viewHolder.contentError.contentErrorAction.setText(R.string.lbl_reload);
```

Comment:
`OverlayErrorNotification` can't handle this specific case, cuz we need to hide 2 different layouts incase of error (It takes only layout to hide in its constructor).

> If we getR.string.action_not_completed or R.string.app_version_unsupported error message from above function than still we should show action button with 'Reload' label. Thoughts ?

Reload seems fine for, as we are giving the user a way to reload the screen irrelevant of the error.

> And we are always showing wifi icon in content_error_icon view:
app:iconName="fa-wifi"

Let me improve this logic.